File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1133.cpp
#include <iostream>

using namespace std;

int main(){
    int x, y;
    cin >> x >> y;
    if(x>y){
        int temp=x;
        x=y;
        y=temp;
    }
    for(int i=x+1;i<y;i++){
        if(i%5==2 or i%5==3){
            cout << i << endl;
        }
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1133.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

x = parseInt(lines.shift())
y = parseInt(lines.shift())

if(x>y){
    temp=x
    x=y
    y=temp
}
for(i=x+1;i<y;i++){
    if(i%5==2 || i%5==3){
        console.log(i)
    }
}